src/main/java/uk/gov/digital/ho/hocs/casework/util/SqsStringMessageAttributeValue.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.digital.ho.hocs.casework.util;
+
```

Comment:
We don't need two of these there is nothing 'sqs' about this and 'sns' about the other

Reply:
The MessageAttributeValue class is actually an import from the specific SQS and SNS library if you check the import. Pain in the backside but I'd rather we have both incase they breaking change one in the future.

Reply:
😢 👍